spec/system/ticket/update/full_quote_header_spec.rb

```diff
@@ -162,7 +196,10 @@ def citation
     end
 
     def name
-      expected.created_by.fullname
+      if expected.origin_by?
+        expected.origin_by.fullname
+      else
+        expected.created_by.fullname
```

Comment:
Something may be missing here :)
 
While at it, it may great to do this in minimalist single-line style. More clear business logic and simplier syntax.

Reply:
Ah alright! Update it now.